doc/getting-started.md

```diff
@@ -114,6 +114,17 @@ MongoDB server version: 5.0.3
 WriteResult({ "nInserted" : 1 })
 ```
 
+To create an admin user from a terminal, provide `is_admin: 1` to the request:
```

Comment:
I think we can simplify the documentation by saying that an admin user needs to be created first, and then additional users can be created with the kernelci-core command line tools.

Ideally we should have all the related changes in kernelci-core merged first, so either we adjust this PR now and wait for kernelci-core changes to be merged or we make a follow-up PR in the documentation later.

Reply:
I think we need to merge this and https://github.com/kernelci/kernelci-core/pull/1210 together as both use URLs from each other's documentation.